diff --git a/lua/lazy/core/loader.lua b/lua/lazy/core/loader.lua
index 9cd5018..2bf841f 100644
--- a/lua/lazy/core/loader.lua
+++ b/lua/lazy/core/loader.lua
@@ -305,7 +305,7 @@ end
 ---@param reason {[string]:string}
 ---@param opts? {force:boolean} when force is true, we skip the cond check
 function M._load(plugin, reason, opts)
-  if not plugin._.installed then
+  if not plugin._.installed  and not ((plugin.pin or Config.options.defaults.pin) and not Config.options.install.missing) then
     return Util.error("Plugin " .. plugin.name .. " is not installed")
   end
 
