diff --git a/lua/lazy/core/loader.lua b/lua/lazy/core/loader.lua
index 2bf841f..c19e05f 100644
--- a/lua/lazy/core/loader.lua
+++ b/lua/lazy/core/loader.lua
@@ -392,7 +392,7 @@ function M.get_main(plugin)
   local normname = Util.normname(plugin.name)
   ---@type string[]
   local mods = {}
-  for _, mod in ipairs(Cache.find("*", { all = true, rtp = false, paths = { plugin.dir } })) do
+  for _, mod in ipairs(Cache.find("*", { all = true, rtp = true, paths = { plugin.dir } })) do
     local modname = mod.modname
     mods[#mods + 1] = modname
     local modnorm = Util.normname(modname)
