diff --git a/lua/nvim-treesitter.lua b/lua/nvim-treesitter.lua
index 963fe730..884b796c 100644
--- a/lua/nvim-treesitter.lua
+++ b/lua/nvim-treesitter.lua
@@ -1,4 +1,3 @@
-local install = require "nvim-treesitter.install"
 local utils = require "nvim-treesitter.utils"
 local info = require "nvim-treesitter.info"
 local configs = require "nvim-treesitter.configs"
@@ -10,7 +9,6 @@ require "nvim-treesitter.query_predicates"
 local M = {}
 
 function M.setup()
-  utils.setup_commands("install", install.commands)
   utils.setup_commands("info", info.commands)
   utils.setup_commands("configs", configs.commands)
   configs.init()

